src/main/java/com/spotify/reaper/resources/CommonTools.java

```diff
@@ -70,25 +70,34 @@ public static RepairRun registerRepairRun(AppContext context, Cluster cluster,
 
     Map<String, RingRange> nodes = getClusterNodes(context, cluster, repairUnit);
     // the next step is to prepare a repair run object
-    RepairRun repairRun = storeNewRepairRun(context, cluster, repairUnit, cause, owner, nodes.keySet().size(),
-                                            repairParallelism, intensity);
-    checkNotNull(repairRun, "failed preparing repair run");
+    segments = repairUnit.getIncrementalRepair() ? nodes.keySet().size() : tokenSegments.size();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "segments". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)